go.mod

```diff
@@ -28,6 +28,7 @@ require (
 	github.com/tendermint/tendermint v0.32.8
 	github.com/tendermint/tm-db v0.2.0
 	gopkg.in/yaml.v2 v2.2.7
+	zombiezen.com/go/capnproto2 v2.18.0+incompatible
```

Comment:
What's up with the two capnproto libs and the `+incompatible` flag?
I looked it up, and see it is a way to deal with libs that predate modules.

https://github.com/capnproto/go-capnproto2 seems to have `go.sum`, but not `go.mod`. If we go with this, seems like a nice PR to clean up those dependenices.